management/azure_mgmt_compute/lib/azure_mgmt_compute.rb

```diff
@@ -2,4 +2,4 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See License.txt in the project root for license information.
 
-require 'generated/azure_mgmt_compute'
\ No newline at end of file
+require '2017-03-30/generated/azure_mgmt_compute'
```

Comment:
👍 👍

Reply:
this is another file I edited manually, I think this file needs to change as we introduce an "uber" client for the gem. We should discuss this.